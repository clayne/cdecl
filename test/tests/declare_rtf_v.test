cdecl @ /dev/null @ -xc++ @ declare f as restricted function @ 0
