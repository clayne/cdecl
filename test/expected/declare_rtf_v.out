void f() __restrict__;
